Public/Src/FrontEnd/UnitTests/Libs/Prelude.IO.ts

```diff
@@ -494,6 +494,11 @@ namespace File {
      */
     export declare function readAllText(file: SourceFile, encoding?: TextEncoding): string;
 
+    /**
+     * Read in a fragment of the graph as a binary file and all all pip from it to the pip graph.
+     */
+    export declare function readPipGraphFragment(name: string, file: SourceFile, dependencyNames: string[]): string;

```

Comment:
>string [](start = 109, length = 6)

What does it return? Also, I think it should be in Transformer namespace, not File namespace

Reply:
Added more detail in transformer version

---
In reply to: [288666141](https://github.com/ljcollins25/BuildXL/pull/1#discussion_r288666141) [](ancestors = 288666141)